crates/sui-indexer/src/models_v2/epoch.rs

```diff
@@ -33,6 +33,39 @@ pub struct StoredEpochInfo {
     pub epoch_commitments: Option<Vec<u8>>,
 }
 
+#[derive(Queryable)]
+pub struct QueryableEpochInfo {
```

Comment:
@stefan-mysten I made separate Queryable* structs instead b/c Stored* is also used on the writing path and we do not want to change that.

To read Queryable struct like QueryableEpochInfo, you can add `.select((<COLUMN LIST>))` in the query building function.

Reply:
@gegaowp I see, so basically with this select `diesel` will know which fields to pick, and this way we avoid getting the `system_state` bytes for the `QueryableEpochInfo`. Am I understanding this correctly?

Reply:
@stefan-mysten we will need to manually `select` the columns, here is an example
https://github.com/MystenLabs/sui/blob/6599250fa828165f00afe294ab72063c0869db25/crates/sui-indexer/src/indexer_reader.rs#L1307